entrypoint.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 set -e
 
+: ${PROSODY_USERID:=101}
+
+if [ "$PROSODY_USERID" != "101" ]; then
+	usermod -u "$PROSODY_USERID" prosody;
+	groupmod -g "$PROSODY_USERID" prosody;
```

Comment:
I don't think you can count on user id being equal to group id. Looks like the later is 102 when I look here.